tests/utils.go

```diff
@@ -1463,17 +1458,26 @@ func GetRunningVirtualMachineInstanceDomainXML(virtClient kubecli.KubevirtClient
 	return stdout, err
 }
 
+func GetVirshCommandWithConnectionParameter(vmi *v1.VirtualMachineInstance) []string {
+	command := []string{"virsh", "-c"}
+	if kutil.IsNonRootVMI(vmi) {
+		return append(command, "qemu+unix:///session?socket=/var/run/libvirt/virtqemud-sock")
+	}
+	return append(command, "qemu+unix:///system?socket=/run/libvirt/virtqemud-sock")
```

Comment:
You're not switching to virtqemud until the next commit, so when this helper is introduced it should still use the libvirtd paths.

Reply:
Thanks so much! Leftover splitting commit :slightly_smiling_face: